src/pages/pad/Pads.tsx

```diff
@@ -1,266 +1,338 @@
-
-                        <SelectSearch
-                            placeholder={Strings.selectMember}
-                            items={patientList}
-                            onChange={(event) => setPatient(event)}
-                            icon={<RiMentalHealthLine />}
-                        />
-                        <div className="flex items-end justify-end mt-2">
-                            <Button onClick={() => handleOnAddPatientToPad()}>{Strings.addMember}</Button>
-                        </div>
-
-                        <List
-                            itemLayout="horizontal"
-                            loading={isListLoading}
-                            dataSource={aditionalMembers}
-                            renderItem={(item, index) => (
-                                <List.Item key={index}>
-                                    <List.Item.Meta
-                                        title={item.label}
-                                    />
-                                </List.Item>
-                            )}
-                        />
-                    </Modal>
-
-
-                </div>
-            }
-        />
-    );
-}
-
-export default Pads;
\ No newline at end of file
+	const [getPads, { isLoading }] = useGetPadsMutation();
+	const [data, setData] = useState<PadDetail[]>([]);
+	const [padList, setPadList] = useState<PadDetail[]>([]);
+	const [branchId, setBranchId] = useSessionStorage(Constants.BRANCH_ID, 0);
+	const [session, setSession] = useSessionStorage(Constants.SESSION_AUTH, 0);
+	const [getPatients] = useGetPatientsMutation();
+	const [getPatientsByBranchOffice] = useGetPatientsByBranchOfficeMutation();
+	const [patientList, setPatientList] = useState<SelectItemOption[]>([]);
+	const [patient, setPatient] = useState<SelectItemOption>();
+	const [aditionalMembers, setAditionalMembers] = useState<SelectItemOption[]>(
+		[]
+	);
+	const [registerAditionalMember] = useRegisterAditionalMemberMutation();
+
+	const [selectedPad, setSelectedPad] = useState<PadDetail>();
+	const [isOpenModal, setIsOpenModal] = useState(false);
+	const [isListLoading, setIsListLoading] = useState(false);
+	const [isActionLoading, setIsActionLoading] = useState(false);
+
+	useEffect(() => {
+		handleGetPads();
+		if (props.rol == UserRoles.ADMIN) {
+			handleGetAllPatients();
+		} else {
+			handleGetPatients();
+		}
+	}, []);
+
+	const [currentPage, setCurrentPage] = useState(1);
+	const cardsPerPage = 20; 
+
+	const indexOfLastCard = currentPage * cardsPerPage;
+	const indexOfFirstCard = indexOfLastCard - cardsPerPage;
+	const currentPads = padList.slice(indexOfFirstCard, indexOfLastCard);
+
+	const handlePageChange = (page: number) => {
+		setCurrentPage(page);
+	};
+
+	const handleGetPads = async () => {
+		try {
+			const response = await getPads({}).unwrap();
+			if (props.rol == UserRoles.RECEPTIONIST) {
+				console.log(`Process rep pads`);
```

Comment:
can you remove this log?

Reply:
Okey, don't worry.
I have already deleted the log

Reply:
Okey, don't worry,